app/transactions/transactions.html

```diff
@@ -18,6 +19,66 @@
         <a href="#"><i class="fa fa-fw fa-user"></i> Log Out</a>
         <a href="#"><i class="fa fa-bell-o"></i> </a>
     </div>
+   
 </nav>
+ <div id="main">
```

Comment:
give the id a more meaningful name or rather use `main` element

Reply:
switched to main element